l1-contracts/src/core/RollupCore.sol

```diff
@@ -126,6 +126,10 @@ contract RollupCore is
     CheatLib.setProtocolContractTreeRoot(_protocolContractTreeRoot);
   }
 
+  function updateManaTarget(uint256 _manaTarget) external override(ITestRollup) onlyOwner {
+    FeeLib.updateManaTarget(_manaTarget);
```

Comment:
We should have an event here. We have been pretty bad generally at adding events when state change, but it is usually good practice since it makes analysis and such much simpler.